### PR TITLE
realization of suggest/postal_unit

### PR DESCRIPTION
**Pull request by gam6itko**
https://dadata.ru/api/suggest/postal_unit/
without deserialization

### CODE REVIEW

**Comment by kstkn** [comment]
Thanks for PR @gam6itko! FYI be aware that `master` branch has 2.* version which is unreleased and actually far away from being released. But if you feel like you want to contribute to 2.* development feel free to do it :+1:

**Review by kstkn — COMMENTED** [review]
I don't get it. You have just introduced that unit test and now excluding it :confused:

**Comment by gam6itko** [comment]
PostalUnitTest useful for testing API request, but it needs TOKEN and SECRET keys. I could not find the public test keys. If you want, i can delete it.

**Comment by kstkn** [comment]
I see. So it's E2E test then, not unit test. Honestly I don't think we need them

File: src/Service/Suggestions/PostalUnit.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        return 'https://suggestions.dadata.ru/suggestions/api/4_1/rs';
+    }
+
+    public function suggest(string $query)
+    {
+        $body = ['query' => $query];
+        $request = $this->apiRequestFactory->createRequest('POST', $this->getBaseUri() . '/suggest/postal_unit', $body);
+        $response = $this->httpClient->sendRequest($request);
+        return json_decode((string) $response->getBody(), true);
+    }
```
Comment (kstkn) [inline src/Service/Suggestions/PostalUnit.php:20]: Why all responses are just decoded into plain array? Our goal is to provide concrete response objects.

See https://github.com/kstkn/dadata/tree/master/src/Model/Response for examples

**Review by kstkn — CHANGES_REQUESTED** [review]

File: src/Service/Suggestions/PostalUnit.php (hunk 1 above)
Comment (gam6itko) [reply src/Service/Suggestions/PostalUnit.php:20]: Ok, sorry for bothering

**Pull request closed without merging**